### PR TITLE
PT-57: created script to add users to Users Table

### PR DESCRIPTION
**Pull request by jmun10**

### CODE REVIEW

File: app/add_user.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        except Exception as e:
+            session.rollback()
+            print(e)
+        finally:
+            session.close()
+    print("User {} added successfully.".format(username))
+    return
+
+
+main()
```
Comment (michardy) [inline app/add_user.py]: To make testing easier could you put this under:
``` python
if __name__ == "__main__":
    main()
```

Comment (jmun10) [reply app/add_user.py]: done

**Comment by michardy** [comment]
It is general best practice to not enter passwords in the command line because the command line is logged to `~/.bash_history`

An alternative is to use the Python password input interface:
``` Python
import getpass

pswd = getpass.getpass('Password:')
```

**Review by timle3 — APPROVED** [review]

**Review by hli086 — APPROVED** [review]
lgtm

**Pull request merged**